Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs

```diff
@@ -92,7 +92,15 @@ private void Update()
     public void Initialize(Material skyboxMat, Light dirLight, AnimationClip skyboxAnimationClip, FeatureFlagsCache featureFlagsCache)
     {
         if (skyboxMat != null)
-            skyboxMaterial = skyboxMat;
+        {
+#if UNITY_EDITOR
+
+            // Create a copy so that the original asset does not get modified by UpdateSkyboxColor. Else
+            // we will get annoying mystery changes in Git.
+            skyboxMat = new Material(skyboxMat);
+#endif
+            skyboxMaterial = new Material(skyboxMat);
```

Comment:
Why does it need double material's creation? It will be assigned once to skyboxMaterial

Reply:
Typo!